src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -2007,6 +2008,20 @@ internal PointerTypeSymbol CreatePointerTypeSymbol(TypeSymbol elementType, Nulla
             throw new NotImplementedException();
         }
 
+        private ImmutableQueue<SourceMethodSymbolWithAttributes> _moduleInitializerMethods = ImmutableQueue<SourceMethodSymbolWithAttributes>.Empty;
+
+        internal void AddModuleInitializerMethod(SourceMethodSymbolWithAttributes method)
```

Comment:
>SourceMethodSymbolWithAttributes [](start = 49, length = 32)

ModuleSymbol?

Reply:
I'm not sure what your suggestion is. MethodSymbol?

Reply:
> I'm not sure what your suggestion is. MethodSymbol?

I am suggesting to remove dependency on specific implementation of a MethodSymbol and simply use MethodSymbol as the parameter type.

---
In reply to: [414529981](https://github.com/dotnet/roslyn/pull/43281#discussion_r414529981) [](ancestors = 414529981)

Reply:
Ah, that's what I thought but I wanted to be sure you didn't actually mean ModuleSymbol.

Reply:
> Ah, that's what I thought but I wanted to be sure you didn't actually mean to do something with a ModuleSymbol.

Sorry for the typo. 

---
In reply to: [414877807](https://github.com/dotnet/roslyn/pull/43281#discussion_r414877807) [](ancestors = 414877807)